src/components/atoms/investors/investors.module.scss

```diff
@@ -0,0 +1,59 @@
+@use "../../../sass/settings";
+@use "../../../sass/typography";
+@use "../../../sass/mixins";
+
+$image-height: 72;
+$image-width: 185;
+
+.investor-logos {
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
+}
+
+.investor-logos__item {
+  @include mixins.padding(24);
+
+  border-top: 1px solid settings.$divider;
+}
+
+.investor-logos__container {
+  @include mixins.constrain-width($image-width, "max");
```

Comment:
I know this was already broken, but this also results in an invalid property value as there are no units for the non-rem value in the mixin